bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -229,11 +229,11 @@ spec:
                       - name: POSTGRES_IMAGE
                         value: registry.redhat.io/rhel8/postgresql-13@sha256:d8ec037e60575b86320fdc162117600240ce53038fb8b9b7296ce525d2b8e605
                       - name: INDEXER_IMAGE
-                        value: quay.io/stolostron/search-indexer:2.7.0-9d6779a859e0935afdabea4ae363bc0277e6b01d
+                        value: quay.io/stolostron/search-indexer:2.7.0-SNAPSHOT-2022-09-27-18-43-55
                       - name: COLLECTOR_IMAGE
-                        value: quay.io/stolostron/search-collector:2.7.0-4f4c8f5770b14b31d16384abb1f5601e0018eb9f
+                        value: quay.io/stolostron/search-collector:2.7.0-SNAPSHOT-2022-09-27-18-43-55
                       - name: API_IMAGE
-                        value: quay.io/stolostron/search-v2-api:2.7.0-7131a1a4707f70a1cf8299177d2ea7a3bc41afee
+                        value: quay.io/stolostron/search-v2-api:2.7.0-SNAPSHOT-2022-09-27-18-43-55
```

Comment:
Do we prefer snapshot or sha?

Reply:
@dislbenn When I ran bundle-image-pickup.sh - it asked for snapshot version - May be that does not translate to sha?

Reply:
@xrajesh it will only ask for the snapshot if you do not run the script within `silent` mode. If you run `./scripts/bundle-image-pickup.sh --silent`, then it should update it based on the latest versions within the `stolostron/pipeline` repo.

Reply:
If you run it without the option, it will ask for manual input.